src/Matterhorn/State/Channels.hs

```diff
@@ -1100,9 +1100,9 @@ toggleChannelFavoriteStatus = do
             Nothing -> trueVal
         pref = Preference 
             { preferenceUserId = myId
-            , preferenceCategory = PreferenceCategoryFavoriteChannelShow
+            , preferenceCategory = PreferenceCategoryFavoriteChannel
```

Comment:
This is an example of a change that belongs in another commit (you had a commit that did a lot of the other renamings of this constructor). How do you stage your commits? If you use your editor, does it have support for staging specific hunks? If you are using `git` on the command line, are you familiar with `git add -p`?